### PR TITLE
Reland unshare

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

**Comment by oliverchang** [comment]
Attempting reland. Diff is https://github.com/google/clusterfuzz/pull/1736/commits/cd662d11a26996d1fff77fa23aa59b7747ae735f

File: src/python/tests/core/bot/fuzzers/libFuzzer/engine_test.py [hunk 1]
```diff
@@ -861,6 +861,23 @@ def mocked_log_error(*args, **kwargs):  # pylint: disable=unused-argument
...
+  def setUp(self):
+    super(UnshareIntegrationTests, self).setUp()
+    os.environ['USE_UNSHARE'] = 'True'
+
+  def compare_arguments(self, target_path, arguments, corpora_or_testcase,
+                        actual):
+    """Compare expected arguments."""
+    self.assertListEqual(
+        actual,
+        ['/usr/bin/unshare', '-U', '--map-root-user', '-n', target_path
```
Comment (inferno-chromium) [inline src/python/tests/core/bot/fuzzers/libFuzzer/engine_test.py:877]: test needs fixing as well

Comment (oliverchang) [reply src/python/tests/core/bot/fuzzers/libFuzzer/engine_test.py]: Done!

**Comment by oliverchang** [comment]
Note that this requires root privileges to run now. Will add sudo support in a future CL if this works.

**Pull request merged**